syncd/syncd.cpp

```diff
@@ -555,6 +560,11 @@ int main(int argc, char **argv)
     getRequest = new swss::ConsumerTable(db, "GETREQUEST");
     getResponse  = new swss::ProducerTable(db, "GETRESPONSE");
 
+#ifdef MLNXSAI
+    std::string mlnx_config_file = "/etc/ssw/ACS-MSN2700/sai_2700.xml";
```

Comment:
yes. that's what i'm thinking about.
later we either parsing a config file or getting the info from cmd. 
I'll also update the configure.ac file later.